cogs/filter.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, bot):
                 mod_info = self.bot.get_channel(259728514914189312)
                 await mod_info.send(
                     f'**{time} | SPAM:** {message.author} has had {user_deleted} '\
-                    f'messages deleted in #welcome-center'
+                    f'messages deleted in #join-logs'

```

Comment:
`mod_info` returns a `Channel` object
You should use mod_info.name to return the name regardless of what it gets changed to